web/components/nav/Navbar.tsx

```diff
@@ -0,0 +1,44 @@
+import { createClient } from '@/utils/supabase/server';
+import { getUserAvatarUrl } from '@utils/lib/common/user_helper';
+import { cookies } from 'next/headers';
+import Image from 'next/image';
+import Link from 'next/link';
+import { StyledLink } from '../common/link';
+import { NotificationPin } from '../notification/NotificationPin';
+import { NavLink } from './NavLink';
+
+export default async function Navbar() {
+  const cookieStore = cookies();
+  const supabase = createClient(cookieStore);
+
+  const {
+    data: { user },
+  } = await supabase.auth.getUser();
+
+  const userAvatarUrl = user ? getUserAvatarUrl(user) : null;
+
+  return (
+    <nav className='flex h-16 w-full max-w-5xl items-center justify-between gap-4 p-3 text-sm'>
+      <Link href='/' className='px-2'>
```

Comment:
Same animate ici sur le logo

```suggestion
      <Link href='/' className='px-2 transform transition-all duration-200 ease-in-out active:scale-[0.9]'>
```